pipelines/rh-advisories/README.md

```diff
@@ -23,6 +23,11 @@ the rh-push-to-registry-redhat-io pipeline.
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | -                                                         |
 
+## Changes in 2.0.0
+* Create and upload product-level SBOMs to the Atlas release instance
+* Update and upload component-level SBOMs to the Atlas release instance
+* The "atlas" field in the RPA specifying which instance to use is mandatory
```

Comment:
This probably doesn't need to be here. The pipeline changelog should most of all say what changed in the pipeline - which is that the three tasks were added. And then perhaps briefly say what they do.